### ISSUE TITLE
Test If everything is working after merging Pull request#14

### ISSUE THREAD

**Issue by Anish-M-code** [Raagul26/Password-Manager-and-Generator--GUI#15]
#14 Pull request has been merged to fix security bugs . @gowtham758550 @Raagul26  i think we need to test this if everything works.
Assigning @Raagul26  for now.  Maybe we should submit this to platforms like up for grabs or open source friday since we dont have time to maintain repos. Any opinion @gowtham758550

**Comment by gowtham758550** [comment]
@Anish-M-code yeah, it's a good idea to use `Up For Grabs` or anything like that
